iosApp/iosApp/iOSApp.swift

```diff
@@ -4,7 +4,7 @@ import Shared
 @main
 struct iOSApp: App {
     init() {
-        KoinKt.doInitKoin()
+        KoinKt.doInitKoin(modules: [])
```

Comment:
I don't know this sample very well, but this seems a bit strange. KoinKt.doInitKoin() is probably a bridging function for a default arg? Why is it not an empty module array?